.ci/scripts/metricbeat-test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+## Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+## or more contributor license agreements. Licensed under the Elastic License;
+## you may not use this file except in compliance with the Elastic License.
+
+set -euxo pipefail
+#
+# Run the functional tests for metricbeat using the functional-test wrapper
+#
+# Parameters:
+#   - STACK_VERSION - that's the version of the stack to be tested. Default '8.0.0-SNAPSHOT'.
+#   - METRICBEAT_VERSION - that's the version of the metricbeat to be tested. Default '8.0.0-SNAPSHOT'.
+#
+
+STACK_VERSION=${1:-'8.0.0-SNAPSHOT'}
+METRICBEAT_VERSION=${2:-'8.0.0-SNAPSHOT'}
```

Comment:
What do we use here? I see it's not pointing to the 6.8-SNAPSHOT in https://github.com/elastic/e2e-testing/pull/717/files#diff-d978f86d9d7e7cdbd67136a16be7eda068c0f2fe5cb38bdadd8aae41a53b845eR21-R22

Reply:
Let's keep 8.0.0. There is a PR #584 that must be merged with 6.8.x after this one, updating the version

Reply:
Good catch, BTW!